Stripe/Checkout/STPCheckoutInternalUIWebViewController.m

```diff
@@ -63,10 +63,10 @@ - (void)viewDidLoad {
     [super viewDidLoad];
 
     if (self.options.logoImage && !self.options.logoURL) {
-        NSURL *url = [NSURL fileURLWithPath:[NSTemporaryDirectory() stringByAppendingPathComponent:[[NSUUID UUID] UUIDString]]];
-        BOOL success = [UIImagePNGRepresentation(self.options.logoImage) writeToURL:url options:0 error:nil];
-        if (success) {
-            self.logoURL = self.options.logoURL = url;
+        NSString *base64 = [UIImagePNGRepresentation(self.options.logoImage) base64Encoding];
```

Comment:
It'll return `nil`, which subsequent code should handle correctly. See https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIKitFunctionReference/#//apple_ref/c/func/UIImagePNGRepresentation